modules/openapi-generator/src/main/resources/JavaJaxRS/spec/bodyParams.mustache

```diff
@@ -1 +1 @@
-{{#isBodyParam}}{{#useBeanValidation}}@Valid {{/useBeanValidation}}{{{dataType}}} {{paramName}}{{/isBodyParam}}
\ No newline at end of file
+{{#isBodyParam}}{{#useBeanValidation}}@Valid {{#required}}@NotNull {{/required}}{{/useBeanValidation}}{{{dataType}}} {{paramName}}{{/isBodyParam}}
```

Comment:
@ruedigerk thanks for the PR. I think we need the `{{^isNulalble}} ... {{/isNullable}}` check as well to ensure the required parameter is not nullable.

Reply:
After thinking about this, I think adding the NotNull annotation for the "not nullable" case is not correct. If the parameter type is not nullable, it is not allowed to pass the value "null" as the parameter value, as far as I understand the spec. But I think it is still allowed to omit the parameter completely (if it is not required). 

So omitting a non-required parameter that is non-nullable still (correctly) results in a null value on the Java side, therefore the NotNull annotation must not be added in this case. Therefore checking only for "required" seems to be the right thing to do to me.

Reply:
But following my own reasoning, the NotNull annotation should no be added if the parameter is required but nullable. Ok, I think I misunderstood your comment and that was what you meant all along. I will change it accordingly.

Reply:
Hello @wing328, I fixed the issue you found. Please review again.